charts/mongodb-kubernetes/values.yaml

```diff
@@ -22,13 +22,14 @@ operator:
   deployment_name: mongodb-kubernetes-operator
 
   # Version of mongodb-kubernetes-operator
-  version: 1.0.0
+  version: 1.0.1
 
   # The Custom Resources that will be watched by the Operator. Needs to be changed if only some of the CRDs are installed
   watchedResources:
   - mongodb
   - opsmanagers
   - mongodbusers
+  - mongodbcommunity
```

Comment:
looks like we forgot this in 1.0.0 - we probably require a change in release notes for this

Reply:
I think it was added later, I didn't add it

Reply:
ah yeah, I see it was added during merging back to master

@SimonBaeumer @mircea-cosbuc we should include this in the release notes. `1.0.0` version has additional bug that it does not watch `MongoDBCommunity` resource as I understand. Is that correct @nammn ?

Reply:
Yes, let's include it in the release notes. I've assumed that our e2e-tests would catch something like this ... how did you noticed it?

Reply:
Updated RN -> https://github.com/mongodb/mongodb-kubernetes/pull/107/files